### ISSUE TITLE
College of the Vestalis

### ISSUE THREAD

**Issue by xirsoi** [xirsoi/ancientreligions#53]
I think that the Vestals could use a little extra oomph.

I propose a titular count title for the Vestalis Maxima, which would allow the other Vestal Virgins to be members of her court, and giver a little bit of prestige.

Other ideas to make the Vestals more interesting?

**Comment by rquinio** [comment]
I was just thinking about something similar !
Is the Vestalis Maxima supposed to retire after 30 years too ? 
In case she dies, the default open elective succession is going to be bad.

Something that could be interesting (from https://en.wikipedia.org/wiki/Vestal_Virgin):
> The Pontifex Maximus, acting as the father of the bride, would typically arrange a marriage with a suitable Roman nobleman. A marriage to a former Vestal was highly honored, and – more importantly in ancient Rome – thought to bring good luck, as well as a comfortable pension.

**Comment by xirsoi** [comment]
Great minds think alike ;)
She is supposed to retire after 30 years, and that was covered by JonSeverinsson when he wrote that part.

That sounds very cool. The Punishment part sounds very neat and flavorful, but difficult to implement. Might be able to do something with the loss of her virginity though.

**Comment by rquinio** [comment]
Yeah, maybe have to see how the AI seduction behaves with vestals (I don't have WoL yet) :)
Also probably a 2nd trait "retired vestal" will be needed due to the cannot_marry = yes on the vestal one.

**Comment by xirsoi** [comment]
There is a retired_vestalis trait already, and it replaces the Vestalis trait when they retire.

**Comment by xirsoi** [comment]
We need to be careful not to break saves with this update.
I wonder if we could make the (proposed) Vestalis Maxima titular county be the de jure vassal of the Pontifex Maximus?

**Comment by xirsoi** [comment]
I've been thinking about it, and the titular county would probably not work well. The Vestals would need to be members of the Vestalis Maxima's court, but then it would screw up the minor titles.

**Comment by xirsoi** [comment]
Events should be created around the prestigious nature of a marriage to a retired Vestal.
Also, if a retired Vestal gets pregnant, the Pontif need to decide whether to pardon her or have her put to death. Ramifications for either decision should be meaningful.

**Comment by xirsoi** [comment]
I've thought about it even more, and my comment on Mar 22 isn't correct.
We could pretty easily code around the Vestalis Maxima to make sure it keeps working.

The assignment of the Vestalis minor titles would be a cinch to update. The only trouble would be handling succession of the c_vestalis_maxima title.

**Comment by xirsoi** [comment]
Everything mentioned thus far would make the Vestals more flavorful/interesting, but wouldn't give them a purpose. Some ideas:

* Vestals were required for religious ceremonies. Having Vestals could enable religious celebrations/decisions.
* Vestal Marriages: Marrying off retired vestals could be a method of opinion management. It should also give prestige and income to the husband.
* May want to consider some kind of cost for maintaining a full College of Vestals
** Income based
** Flat cost for investing each Vestal.
* Moral Authority: Each Vestal granting +2% MA?
* Perhaps Vestals are required for certain laws.
* Be able to donate money to the Vestals for piety?
* Give land to a vestal for _lots_ of piety?
* Vestals had expansive rights and privileges. Flavor events should be written for them.
* The maintenance of the Vestals was considered a blessing on the Roman state. Some kind of morale penalties or revolt risk could be put in place if they don't exist or the roster isn't full.

**Comment by xirsoi** [comment]
Something needs to be done to make the Vestals not completely boring before 1.0, but more detailed/full improvements can wait until 2.0

**Comment by xirsoi** [comment]
Give the Pontifex Maximus more options to directly appoint vestals, instead of having to wait until someone nominates a child.

**Comment by xirsoi** [comment]
Vestals as Councilors?

Other realms had Vestals as well. One Vestal for a duchy, another for Kings, one for Empires, one for the Roman Empire, one if you have the Pontifex in your empire, and one if you hold Rome itself. That makes a total of 6 possible Vestals for a restored Roman Empire with the Pontificate and c_roma. The Pontificate Vestal should be the Vestalis Maxima.

**Comment by xirsoi** [comment]
The college as a society could also work. Vestals in other realms becomes slightly harder at that point though.
